TACC-Cloud/hazmapper
# Task/DES-2131: Fix linting and testing and add github ci.

## Overview: ##
Fix linting and testing and added github actions files.

## PR Status: ##


## Related Jira tickets: ##

* [DES-2131](https://jira.tacc.utexas.edu/browse/DES-2131)

## Summary of Changes: ##
Added packages `tslint-config-prettier`, `stylelint`, `stylelint-stylus` for prettier and css support respectively.
Added `npm run lint:js` and `npm run lint:css` npm scripts.
Added a `npm run start:local` as well for convenience.
Fixed `npm run test` script so that it doesn't watch for changes instead of finishing.
Added github actions file for running ci scripts.
Added results from running linters automatically fixing.
Added instructions for the new options into `README.md`.

## Testing Steps: ##
1. Run `npm run lint` and ensure that everything works.
2. Run `npm run test` and ensure that everything works.
3. Run hazmapper locally to ensure that everything works as they did.

## UI Photos:

## Notes: ##
Will have a part 2 PR for switching to eslint from deprecated tslint.
